package.json

```diff
@@ -121,6 +121,11 @@
                                 "type": "string",
                                 "description": "The location of the generated JavaScript code (the bundle file). Normally this should be \"${workspaceRoot}/.vscode/.react\"",
                                 "default": null
+                            },
+                            "iosRelativeProjectPath": {
+                                "type": "string",
+                                "description": "Relative path to the ios/ folder, if it is not located on the project root.",
+                                "default": null
```

Comment:
The default in this case would be "ios" (That what we'll use if they don't provide this property).